modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java

```diff
@@ -547,6 +551,8 @@ public CodegenOperation fromOperation(String path, String httpMethod, Operation
                     consumesXml = true;
                 } else if (isMimetypePlainText(mimeType)) {
                     consumesPlainText = true;
+                } else if (isMimetypeHtmlText(mimeType)) {
```

Comment:
Consuming html plaintext may not make sense (I don't know what is supposed to happen), we can adjust this.